CMakeLists.txt

```
@@ -237,7 +238,7 @@ set(INTERFACE_HEADERS
         ${PROJECT_SOURCE_DIR}/include/roboteam_ai/interface/widgets/ManualControlWidget.h
         ${PROJECT_SOURCE_DIR}/include/roboteam_ai/interface/widgets/PlaysWidget.hpp
         ${PROJECT_SOURCE_DIR}/include/roboteam_ai/interface/widgets/GraphWidget.h
-        )
+        include/roboteam_ai/stp/skills/TestSkill.h)
```

Please try to use absolute paths with the help of variables in CMake, so if a folder structure ever changes, the CMakeLists files can be easily updated
```suggestion
        ${PROJECT_SOURCE_DIR}/include/roboteam_ai/stp/skills/TestSkill.h)
```

---

Good catch, that include did not belong there, not sure how it got there but it's gone now.